src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -1,27 +1,24 @@
 // @ts-check
-import React, {
-  useContext,
-  useState,
-  useEffect,
-  useCallback,
-  useRef,
-} from 'react';
+import React, { useCallback, useContext, useMemo, useRef } from 'react';
 import { Virtuoso } from 'react-virtuoso';
-
-import { smartRender } from '../../utils';
-import MessageNotification from './MessageNotification';
 import { ChannelContext, TranslationContext } from '../../context';
+import { smartRender } from '../../utils';
+import { EmptyStateIndicator as DefaultEmptyStateIndicator } from '../EmptyStateIndicator';
 import { EventComponent } from '../EventComponent';
 import { LoadingIndicator as DefaultLoadingIndicator } from '../Loading';
-import { EmptyStateIndicator as DefaultEmptyStateIndicator } from '../EmptyStateIndicator';
 import {
-  FixedHeightMessage,
+  FixedHeightMessage as DefaultMessage,
   MessageDeleted as DefaultMessageDeleted,
 } from '../Message';
+import { useNewMessageNotification } from './hooks/useNewMessageNotification';
+import { usePrependedMessagesCount } from './hooks/usePrependMessagesCount';
+import { useShouldForceScrollToBottom } from './hooks/useShouldForceScrollToBottom';
+import MessageNotification from './MessageNotification';
+
+const prependOffset = 10000;
```

Comment:
should we increase this to 50k just to be on the safe side? what happens if we prepend more than 10k?